targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_ARM_STD/rtl8195a.sct

```diff
@@ -29,9 +29,6 @@ LR_IRAM 0x10007000 (0x70000 - 0x7000) {
     *libc.a (+RW)
     *(.sdram.data*)
     *lib_peripheral_mbed_arm.ar (+RW)
-  }
-
-  RW_IRAM2 +0 UNINIT FIXED {
```

Comment:
Is the main thing that this PR is doing is replacing RAM2 block usage with the RAM1 block?

Reply:
There is some part of the memory at the end of RAM2 which contains data that we donot want to touch throughout the lifecycle of the program. RAM2 block usage might create a scenario where the memory might reach the addresses that we donot wish to touch, hence we have replaced RAM2 block usage with RAM1.